src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_TypeInference.cs

```diff
@@ -1733,10 +1714,7 @@ static void F(dynamic x, object y)
                 source,
                 references: new[] { ValueTupleRef, SystemRuntimeFacadeRef },
                 parseOptions: TestOptions.Regular8);
-            comp.VerifyDiagnostics(
-                // (13,13): warning CS8620: Nullability of reference types in argument of type '(dynamic x, object y)' doesn't match target type '(dynamic, object?)' for parameter 't' in '(dynamic, object?) E.F<(dynamic, object?)>(C<(dynamic, object?)> c, (dynamic, object?) t)'.
```

Comment:
>// (13,13): warning CS8620: Nullability of reference types in argument of type '(dynamic x, object y)' doesn't match target type '(dynamic, object?)' for parameter 't' in '(dynamic, object?) E.F<(dynamic, object?)>(C<(dynamic, object?)> c,  [](start = 16, length = 240)

It is not clear why this warning is no longer reported. Are we inferring nullability per tuple element?

Reply:
`(dynamic, object)` is implicitly convertible to `(dynamic, object?)`.

---
In reply to: [181220443](https://github.com/dotnet/roslyn/pull/26107#discussion_r181220443) [](ancestors = 181220443)